From 2b3569f4562129d07a24ef6621263d0f326a816b Mon Sep 17 00:00:00 2001
From: don bright <hmbright@fastmail.com>
Date: Sat, 29 Apr 2023 22:33:13 -0500
Subject: [PATCH] hwdb: add hardware rfkill key for Dell Latitude E6* models
 (#27462)

Hello

This pull req is adapting pull req #5772 (which fixed issue #5047), for the very similar computer Dell Latitude E6420 which has the same problem with the hardware switch to toggle wifi (aka rfkill). The symptom is the following repeated msgs in dmesg

[  309.010284] atkbd serio0: Use 'setkeycodes e008 <keycode>' to make it known.
[  309.016020] atkbd serio0: Unknown key pressed (translated set 2, code 0x88 on isa0060/serio0).

Adding this line to include E6 models causes these messages to stop showing in dmesg

Thank you

(cherry picked from commit 46b8c3f5b297ac034f2d024c1f3d84ad2c17f410)

Related: RHEL-5950
---
 hwdb.d/60-keyboard.hwdb | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/hwdb.d/60-keyboard.hwdb b/hwdb.d/60-keyboard.hwdb
index 82f3aaf051..4b470cf18d 100644
--- a/hwdb.d/60-keyboard.hwdb
+++ b/hwdb.d/60-keyboard.hwdb
@@ -398,7 +398,8 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnDell*:pnPrecision*:*
  KEYBOARD_KEY_88=!                                      # wireless switch
  KEYBOARD_KEY_9e=!f21
 
-# Dell Latitude E7*
+# Dell Latitude E[67]*
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svnDell*:pnLatitude*E6*:*
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnDell*:pnLatitude*E7*:*
  KEYBOARD_KEY_88=unknown                                # Fn-PrtScr rfkill - handled in HW
 
